src/AccordionItem/index.jsx

```diff
@@ -24,6 +24,11 @@ export default class AccordionItem extends Component {
 
   componentDidMount() {
     this.setMaxHeight();
+    // allow overflow for absolute positioned elements inside
+    // the item body, but only after animation is complete
+    ReactDOM.findDOMNode(this).addEventListener('transitionend', () => {
+      if (this.props.expanded) this.allowOverflow();
```

Comment:
Do we need this workaround anymore?

Just checked React15 for [this](https://github.com/facebook/react/pull/6005/files#diff-8949552676e903f486caef00b60d5630R291)